modules/backend/models/Preference.php

```diff
@@ -178,45 +178,45 @@ protected function getFallbackLocale($locale)
     public function getLocaleOptions()
     {
         $localeOptions = [
-            'ar' => [Lang::get('system::lang.locale.ar'), 'flag-sa'],
```

Comment:
Is there anyway CSS trickery that we could do in order to retain the existing `flag-locale` class syntax instead of requiring the `flag-icon` addition?

Something like:
```css
*[class*="flag-"] {
     .flag-icon()
}
```

Reply:
I'm sorry I have no idea what that css you wrote do, I'm not a css expert at all (I'm a backend developer...)

Reply:
`*` targets all elements, `[]` is an attribute selector, `*=` means value is included anywhere (as opposed to `^=` starting with and `$=` ending with).